web-common/src/lib/event-bus/events.ts

```diff
@@ -16,6 +16,17 @@ interface NotificationOptions {
 }
 
 export interface BannerMessage {
+  type: "default" | "success" | "info" | "warning" | "error";
   message: string;
   includesHtml?: boolean;
+
+  iconType: "none" | "alert" | "check" | "sleep" | "loading";
```

Comment:
I think the icon should primarily be a derivative of the `type`. For example, by default, an `info` type should have an `info` icon, and the `warning` and `error` types should have an `alert` icon. Otherwise, we'll risk inconsistency. But we could have an optional `iconOverride` to handle the special cases.

Reply:
These are meant to map to the icon name. Alert icon is actually the one used the most. Since there is enough usages that breaks these defaults, there is no point in defaults.